code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -399,8 +401,18 @@
 	if(!C.vampire_affected(mind))
 		C.visible_message("<span class='warning'>[C] seems to resist the takeover!</span>", "<span class='notice'>Your faith of [ticker.Bible_deity_name] has kept your mind clear of all evil</span>")
 	if(!ishuman(C))
-		to_chat(src, "<span class='warning'>You can only enthrall humanoids!")
+		to_chat(src, "<span class='warning'>You can only enthrall humanoids!</span>")
 		return 0
+	if(ishuman(src))
```

Comment:
Did you copy this block, because if you did you should make it a proc a la can_suck(user, target) or something of the sort.